applications/teletype/priv/templates/cnam_request.html

```diff
@@ -67,7 +67,7 @@ <h3 style="margin:0;padding:0;font-family:'Open Sans',sans-serif;color:#555555;f
                             </tr>
                             <tr>
                                 <td style="padding:10px;background-color:#f5f5f5;text-align:right;font-family:'Open Sans',sans-serif;color:#555555;font-size:13px;font-weight:400;line-height:20px;border-size:3px;border-color:#ffffff;border-right-style:solid;border-bottom-style:solid;" width="30%"><b>Number State</b></td>
-                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.state}}</td>
+                                <td style="padding:10px;background-color:#eaeaea;font-size:13px;font-family:'Open Sans',sans-serif;color:#555555;line-height:20px;border-size:3px;border-color:#ffffff;border-bottom-style:solid;" width="70%">{{request.number_state}}</td>

```

Comment:
Number_state is the field name for notify.

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_cnam_notifier.erl#L150

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_telnyx_cnam.erl#L155

https://github.com/2600hz/kazoo/blob/master/core/kazoo_number_manager/src/providers/knm_vitelity_cnam.erl#L338